### ISSUE TITLE
Training for anymal_c_rough

### ISSUE THREAD

**Issue by andyyounes1** [leggedrobotics/legged_gym#78]
So anymal_c_flat trains perfectly, but i have issues training anymal_c_rough:
python ./legged_gym/scripts/train.py --task=anymal_c_rough
Importing module 'gym_38' (/home/andyyounes/isaacgym/python/isaacgym/_bindings/linux-x86_64/gym_38.so)
Setting GYM_USD_PLUG_INFO_PATH to /home/andyyounes/isaacgym/python/isaacgym/_bindings/linux-x86_64/usd/plugInfo.json
PyTorch version 2.4.1+cu121
Device count 1
/home/andyyounes/isaacgym/python/isaacgym/_bindings/src/gymtorch
Using /home/andyyounes/.cache/torch_extensions/py38_cu121 as PyTorch extensions root...
Emitting ninja build file /home/andyyounes/.cache/torch_extensions/py38_cu121/gymtorch/build.ninja...
Building extension module gymtorch...
Allowing ninja to set a default number of workers... (overridable by setting the environment variable MAX_JOBS=N)
ninja: no work to do.
Loading extension module gymtorch...
Setting seed: 1
Not connected to PVD
+++ Using GPU PhysX
Physics Engine: PhysX
Physics Device: cuda:0
GPU Pipeline: enabled
Segmentation fault (core dumped)

Someone please help me
